projects/plugins/boost/readme.txt

```diff
@@ -165,52 +165,71 @@ If you run into compatibility issues, please do let us know. You can drop us a l
+
+## 1.0.6 - 2021-05-25
+
+- Fix: Failed to execute 'json' errors
+- Fix: Connection UI Border issues
+- Update: Improve Jetpack compatibility
+- Update: Improve Critical CSS Compatibility with caching and minification plugins
+- Update: Clean up JavaScript dependencies
+
+## 1.0.5 - 2021-05-13
+
+- Fixed: Defer JavaScript compatibility with XML Requests
+
+## 1.0.4 - 2021-05-06
+
+- Fixed: Web Stories compatibility
+- Improved: "Defer Non-Essential Javascript" module compatibility with other plugins
+
+## 1.0.3 - 2021-04-26
+
+- Updated: Support for AMP Plugin 2.0+
+- Updated: No longer defer JavaScript on POST, AJAX, Cron requests and sitemaps.
+
+## 1.0.2 - 2021-04-22
+
+- Improved: HTML Media tag handling
+- Fixed: Metrics timeout caused by caching in the REST API
+
+## 1.0.1 - 2021-04-20
+
+- Fixed: An issue where the connection iframe would sometimes break
+- Updated: On connection: showing an XML RPC Error instead of HTTP 500 when XML-RPC is disabled
+
+## 1.0.0 - 2021-04-19
 
-= 1.1.0 =
-* Update: User connection is no longer required for Speed Scores.
-* Update: Completely revamped how site speed scores are retreived.
-* Update: Reduced backend dashboard JavaScript bundle size.
-* Update: Added a message to explain how site score is calculated.
-* Update: Added "Offline Mode" to allow testing Jetpack Boost on local environments easily.
-* Update: Improved error handling and the error messages provided.
-* Update: Improved Critical CSS Generation stability.
-* Update: Remove animations from Critical CSS.
-* Fix: Incompatibility with UsersWP and similar plugins that might introduce redirects during Critical CSS Generation.
-
-= 1.0.6 =
-* Fix: Failed to execute 'json' errors
-* Fix: Connection UI Border issues
-* Update: Improve Jetpack compatibility
-* Update: Improve Critical CSS Compatibility with caching and minification plugins
-* Update: Clean up JavaScript dependencies
-
-= 1.0.5 =
-* Fixed: Defer JavaScript compatibility with XML Requests
-
-= 1.0.4 =
-* Fixed: Web Stories compatibility
-* Improved: "Defer Non-Essential Javascript" module compatibility with other plugins
-
-= 1.0.3 =
-* Updated: Support for AMP Plugin 2.0+
-* Updated: No longer defer JavaScript on POST, AJAX, Cron requests and sitemaps.
-
-= 1.0.2 =
-* Improved: HTML Media tag handling
-* Fixed: Metrics timeout caused by caching in the REST API
-
-= 1.0.1 =
-* Fixed: An issue where the connection iframe would sometimes break
-* Updated: On connection: showing an XML RPC Error instead of HTTP 500 when XML-RPC is disabled
-
-= 1.0.0 =
 - This update brings a lot of stability improvements.
 - We've been hard at work to get here and Jetpack Boost v1.0.0 is finally here! 🎉
 
-= 0.9.19 =
+## 0.9.19 - 2021-03-19
 
 - We've refactored the plugin quite a bit, starting from the UI to stability fixes.
 
-= 0.9.1 =
+## 0.9.1 - 2020-12-29
 
-- First public alpha release
+- First public alpha release
```

Comment:
In the future, I think it would be fine to only keep the latest major in the `readme.txt` file. The more things you have in here, the more work translators have when they want to translate the readme on translate.wordpress.org. I think it makes sense to allow them to only focus on the latest changelog entries, since those are the ones you want them to translate.

At the end of that readme.txt, you can add a link to the full changelog.md file that lives on GitHub for folks who want to see the full, historical changelog. (That md file does not get offered for translation on translate.wordpress.org).

Check how we do it for Jetpack here:
https://github.com/Automattic/jetpack/blob/94ed69c2209934af010910e8ccb0e8c38cb7178b/projects/plugins/jetpack/readme.txt#L269-L271

Reply:
Nice! I've made this change now so that I don't need to remember in future releases. At least until we hit a 2.0 ;)